### PR TITLE
Auto-updating README with crates.io versions, moved NATS comment

### PR DESCRIPTION
**Pull request by brooksmtownsend**
Apparently, our script to grab the capability contract ID does not handle comments on the same line, so I moved the NATS comment up a line.

The main feature of this PR is `crates.io` badges for the latest versions in the readme, this ensures anyone who visits this repository can automatically see the latest published crate version. I threw in a snippet about the latest OCI tag, but the OCI reference is there at least without the tag. Curious if there are suggested improvements.

I've also trawled through our READMEs and removed some lingering `wascc` references, added crates.io badges, and standardized the format of the READMEs. This will make the crates.io page look similar for all the providers and show accurate information.

### CODE REVIEW

**Comment by brooksmtownsend** [comment]
Definitely would recommend taking a look at https://github.com/wasmCloud/capability-providers/tree/fd3c2d6a32e5997dbcbb93a8388091b8c3a71c11#latest-versions to see what the README looks like live.

**Comment by autodidaddict** [comment]
Absolutely love the way this new readme looks. :+1:

File: http-server/README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-[![crates.io](https://img.shields.io/crates/v/wascc-httpsrv.svg)](https://crates.io/crates/wascc-httpsrv)&nbsp;
-![Rust](https://github.com/wascc/http-server-provider/workflows/Rust/badge.svg)
-![license](https://img.shields.io/crates/l/wascc-httpsrv.svg)&nbsp;
-[![documentation](https://docs.rs/wascc-httpsrv/badge.svg)](https://docs.rs/wascc-httpsrv)
+[![crates.io](https://img.shields.io/crates/v/wasmcloud-httpserver.svg)](https://crates.io/crates/wasmcloud-httpserver)&nbsp;
+![Rust](https://github.com/wasmcloud/capability-providers/workflows/HTTPSERVER/badge.svg)
```
Comment (autodidaddict) [inline http-server/README.md:2]: Is this supposed to be all caps? I don't know if github URLs are case sensitive...

File: http-server/README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+# wasmCloud HTTP Server Provider
 
 This library is a _native capability provider_ for the `wasmcloud:httpserver` capability. Only actors signed with tokens containing this capability privilege will be allowed to use it. 
 
-It should be compiled as a native shared object binary (linux `.so`, mac `.dylib`, windows `.dll`) and made available to the **waSCC** host runtime as a plugin. If you want to statically compile (embed) it into a custom waSCC host, then simply enable the `static_plugin` feature in your dependencies:
+It should be compiled as a native shared object binary (linux `.so`, mac `.dylib`, windows `.dll`) and made available to the **wasmCloud** host runtime as a plugin. If you want to statically compile (embed) it into a custom wasmCloud host, then simply enable the `static_plugin` feature in your dependencies:
 
 ```
-wascc-httpsrv = { version = "0.9.0", features = ["static_plugin"] }
+wasmCloud-httpserver = { version = "0.11.1", features = ["static_plugin"] }
```
Comment (autodidaddict) [inline http-server/README.md:13]: should be `wasmcloud-httpserver`

File: logging/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [![crates.io](https://img.shields.io/crates/v/wasmcloud-logging.svg)](https://crates.io/crates/wasmcloud-logging)&nbsp;
-![Rust](https://github.com/wasmcloud/logging-provider/workflows/Rust/badge.svg)&nbsp;
+![Rust](https://github.com/wasmcloud/capability-providers/workflows/LOGGING/badge.svg)&nbsp;
```
Comment (autodidaddict) [inline logging/README.md:2]: more capslock?

File: nats/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-nats"
-version = "0.10.0"
+version = "0.10.1"
 authors = ["Kevin Hoffman <alothien@gmail.com>"]
```
Comment (autodidaddict) [inline nats/Cargo.toml:4]: Let's change this to `wasmCloud team` like the other crates.

**Review by autodidaddict — CHANGES_REQUESTED** [review]
Mostly just looks like issues with capslock search/replace

File: http-server/README.md (hunk 1 above)
Comment (brooksmtownsend) [reply http-server/README.md:2]: Funny enough, the workflow URLS _are_ case sensitive, so they have to be all caps (which is what our workflows are)

Comment (autodidaddict) [reply http-server/README.md:2]: Hah. Did not expect that.

File: logging/README.md (hunk 3 above)
Comment (brooksmtownsend) [reply logging/README.md:2]: Same comment as above

**Review by autodidaddict — APPROVED** [review]

**Pull request merged**